tests/e2e/downloadExcel.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { expect, test } from "@playwright/test";
+import waitForNewScenarioPage from "~/tests/e2e/helpers/waitForNewScenarioPage";
+
+test("can download Excel file for scenario results", async ({ page, baseURL }) => {
+  await waitForNewScenarioPage(page, baseURL);
+  // Run scenario with default parameters
+  await page.click('button:has-text("Run")');
+  await page.waitForURL(new RegExp(`${baseURL}/scenarios/[a-f0-9]{32}`));
+  await page.waitForSelector("#btn-download-excel");
+  const downloadBtn = page.locator("#btn-download-excel");
+  await expect(downloadBtn).toBeVisible();
+  await expect(page.getByText("Prevalence").first()).toBeVisible(); // Wait for data
+
+  const [download] = await Promise.all([
+    // Start waiting for the download
+    page.waitForEvent("download"),
```

Comment:
This was a useful pattern I found online last time I wrote a Playwright test for handling download. Iirc, because both clicking on the download button and waiting for the download event are asynchronous, and triggering the former means the latter can complete any time after that, listing them both together in a Promise.all and waiting for them both to complete is quite a neat way of ensuring you don't miss the event. `download` is the value returned by the first promise in the list, which we can then do further checks on about the download process.